apps/showcase/tsconfig.spec.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.jest",
   "compilerOptions": {
     "outDir": "test",
-    "rootDir": "."
+    "rootDir": ".",
+    "noUnusedParameters": false
```

Comment:
should this be removed ?

Reply:
Added on purpose to have the test running else it raise the error that "dictionnaries" is not used in the reviver
I added this rule only for test I think it's okay